stellar-core/debian/rules

```diff
@@ -13,19 +13,18 @@ include /usr/share/dpkg/buildflags.mk
 # package maintainers to prepend CXXFLAGS
 export DEB_CXXFLAGS_MAINT_PREPEND  = -stdlib=libc++ -fno-omit-frame-pointer -isystem /usr/include/libcxxabi
 
-# set DEB_CONFIGURE_OPTS in your environment to override
-DEB_CONFIGURE_OPTS ?= --prefix=/usr --includedir=/usr/include --mandir=/usr/share/man --infodir=/usr/share/info --sysconfdir=/etc --localstatedir=/var
+# set EXTRA_DEB_CONFIGURE_OPTS in your environment to append options
 
 %:
-	dh $@ --with autoreconf --with systemd
+	dh $@ --with autoreconf --with systemd --parallel
 override_dh_autoreconf:
 	./autogen.sh --skip-submodules # This skips the autogen.sh `git submodule update --init`
 
 
 # dh_make generated override targets
 # This is example for Cmake (See https://bugs.debian.org/641051 )
 override_dh_auto_configure:
-	CC=clang-8 CXX=clang++-8 ./configure $(DEB_CONFIGURE_OPTS)
+	CC=clang-8 CXX=clang++-8 ./configure --prefix=/usr --includedir=/usr/include --mandir=/usr/share/man --infodir=/usr/share/info --sysconfdir=/etc --localstatedir=/var $(EXTRA_DEB_CONFIGURE_OPTS)
```

Comment:
Debian helper environment variables are by convention all prefixed with `DEB_`, the dpkg-buildflags tool uses _PREPEND / _APPEND suffixes such as `DEB_CXXFLAGS_MAINT_APPEND` to append or prepend parameters,  although we are not using dpkg-buildflags here would it make sense to use a similar environment variable name ?

If naming choice is intentional we should also update the relevant documentation found at https://github.com/stellar/packages/blob/master/docs/building-packages.md#deb_configure_opts

Reply:
I thought it would be better to keep it different to avoid potential confusion. The variable is stellar and stellar-core specific and by calling it something like `DEB_CONFIGURE_APPEND` we will make it look like it's an official debian variable.